<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogInputError</class>
 <widget class="QDialog" name="DialogInputError">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Input Error</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_location">
     <item>
      <widget class="QLabel" name="label_file">
       <property name="text">
        <string>In file </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_filename">
       <property name="text">
        <string>filename</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_line">
       <property name="text">
        <string>,  line </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_lineno">
       <property name="text">
        <string>1234</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_location">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_error">
     <item>
      <widget class="QLabel" name="label_error">
       <property name="text">
        <string>The following value is out of range: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_errValue">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_hash">
       <property name="text">
        <string>#</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_description">
       <property name="text">
        <string>Forecast loop control #4</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_error">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_newvalue">
     <item>
      <widget class="QLabel" name="label_preface">
       <property name="text">
        <string>Please input a new value in the range</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_minValue">
       <property name="text">
        <string>1234</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_to">
       <property name="text">
        <string>to</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_maxValue">
       <property name="text">
        <string>1234</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_newValue">
       <property name="text">
        <string>4</string>
       </property>
       <property name="maxLength">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_newvalue">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Abort|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogInputError</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogInputError</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
